test/Interop/Cxx/class/invalid-unsafe-projection-errors.swift

```diff
@@ -30,6 +30,7 @@ public func test(x: M) {
   x.test2()
   // CHECK: note: C++ method 'test3' that returns unsafe projection of type 'Ptr' not imported
   x.test3()
-  // CHECK: note: C++ method 'begin' that returns an unsafe iterator not imported: use Swift Sequence APIs instead
+  // CHECK: error: value of type 'M' has no member 'begin'
```

Comment:
Here, the error message is "value of type 'M' has no member 'begin'"  but the error message in `DiagnosticsSema.def` reads as  "type %0 has no member %1" . Can you create a small test code that generates "type %0 has no member %1" error message? I think it good to add that test here as well in addition to this.